components/camel-ftp/src/test/java/org/apache/camel/component/file/remote/FromFileToFtpWithChmodTest.java

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
The license header in the top of the file should be ASF standard license we use, so you can copy it from another file.